### PR TITLE
k8s contrib dependancy

### PR DESCRIPTION
**Pull request by abhinandanpb**
1) Pulling contrib if not present in local dir.
2)  updating to latest netplugin version

### CODE REVIEW

File: vagrant/k8s/setup_cluster.sh [hunk 1]
```diff
@@ -48,15 +48,28 @@ function GetContiv {
...
+  	fi
+	popd
+}
+
 # kubernetes version to use -- defaults to v1.2.3
 : ${k8sVer:=v1.2.3}
 
 # contiv version
-: ${contivVer:=v0.1-05-05-2016.18-38-56.UTC}
+: ${contivVer:=v0.1-09-08-2016.20-56-40.UTC}
```
Comment (gaurav-dalvi) [inline vagrant/k8s/setup_cluster.sh]: Can we make this env variable or something ?
Right now its hard coded.

**Comment by gaurav-dalvi** [comment]
LGTM

File: vagrant/k8s/setup_cluster.sh (hunk 1 above)
Comment (abhinandanpb) [inline vagrant/k8s/setup_cluster.sh]: no sure what you mean by its hard coded. This is the "env variable". User can export contivVer to any version of his/her choice. If its not exported then it defaults to the above version.

Comment (gaurav-dalvi) [inline vagrant/k8s/setup_cluster.sh]: Got it. Thanks ! Usually after some time we might need to update this default version. We have been facing some issues in net_demo_installer script as it has some old hard coded version but you can set that env variable and change it to new version.

Anyways this looks good then !

**Pull request merged**